persistence/remote/src/test/resources/remote-cl-uri-config.xml

```diff
@@ -1,33 +1,32 @@
                           urn:infinispan:config:store:remote:${infinispan.core.schema.version} https://infinispan.org/schemas/infinispan-cachestore-remote-config-${infinispan.core.schema.version}.xsd"
-      xmlns="urn:infinispan:config:${infinispan.core.schema.version}"
-      xmlns:remote="urn:infinispan:config:store:remote:${infinispan.core.schema.version}" >
+	xmlns="urn:infinispan:config:${infinispan.core.schema.version}"
+	xmlns:remote="urn:infinispan:config:store:remote:${infinispan.core.schema.version}">
 
-   <!-- Default cache named to preserve old default cache name -->
-   <cache-container default-cache="RemoteStoreConfigUriTest">
-      <local-cache name="RemoteStoreConfigUriTest">
-         <persistence passivation="false">
-            <remote-store
-	xmlns="urn:infinispan:config:store:remote:${infinispan.core.schema.version}"
-	cache="RemoteStoreConfigUriTest" fetch-state="false" preload="false"
-	purge="false" shared="true" read-only="false" hotrod-wrapping="false"
-	raw-values="false" socket-timeout="60000" tcp-no-delay="true"
-	balancing-strategy="org.infinispan.client.hotrod.impl.transport.tcp.RoundRobinBalancingStrategy"
-	transport-factory="org.infinispan.client.hotrod.impl.transport.netty.ChannelFactory"
-	key-size-estimate="62" value-size-estimate="512"
-	force-return-values="false" segmented="false"
-	uri="hotrods://127.0.0.1:19711?use_ssl=false&amp;tcp_no_delay=true&amp;tcp_keep_alive=false&amp;force_return_values=false">
-               <connection-pool max-active="99" max-pending-requests="97" min-idle="98"/>
-               <async-executor factory="org.infinispan.client.hotrod.impl.async.DefaultAsyncExecutorFactory">
-                  <property name="pool_size">1</property>
-                  <property name="queue_size">10000</property>
-               </async-executor>
-            </remote-store>
-         </persistence>
-      </local-cache>
-   </cache-container>
+	<!-- Default cache named to preserve old default cache name -->
+	<cache-container
+		default-cache="RemoteStoreConfigUriTest">
+		<local-cache name="RemoteStoreConfigUriTest">
+			<persistence>
+				<remote-store
+					xmlns="urn:infinispan:config:store:remote:${infinispan.core.schema.version}"
+					cache="RemoteStoreConfigUriTest" fetch-state="false"
+					preload="false" purge="false" shared="true" read-only="false"
+					hotrod-wrapping="false" raw-values="false" socket-timeout="60000"
+					tcp-no-delay="true"
+					balancing-strategy="org.infinispan.client.hotrod.impl.transport.tcp.RoundRobinBalancingStrategy"
+					transport-factory="org.infinispan.client.hotrod.impl.transport.netty.ChannelFactory"
+					key-size-estimate="62" value-size-estimate="512"
+					force-return-values="false" segmented="false"
+					uri="hotrods://127.0.0.1:19711?use_ssl=false&amp;tcp_no_delay=true&amp;tcp_keep_alive=false&amp;force_return_values=false">
+					<connection-pool max-active="99"
+						max-pending-requests="97" min-idle="98" />
+				</remote-store>
```

Comment:
This can still be simplified. I think the following is the minimum:

```xml
<remote-store xmlns="urn:infinispan:config:store:remote:${infinispan.core.schema.version}"
cache="RemoteStoreConfigUriTest"
shared="true"
uri="hotrods://127.0.0.1:19711?use_ssl=false&amp;tcp_no_delay=true&amp;tcp_keep_alive=false&amp;force_return_values=false" />

```

Reply:
The `fetch-state` attributes can be removed as the default value is `false`.

The segmented default is `true`, however for the sake of this test setting this to `true | false` has no impact, so we can also remove this.

The `default-cache` attribute can also be removed on the `cache-container` element.